project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -39,20 +46,22 @@
 
     /** In this radius around "LOCATION" places will be searched. */
     private static final int SEARCH_RADIUS = 5000;
-    // TODO(M1): check at least 10 results, and if less extend radius
 
     /** The entry point for a Google GEO API request. */
     private static final GeoApiContext CONTEXT = new GeoApiContext.Builder()
         .apiKey(System.getenv("API_KEY"))
         .build();
 
+    /** A mapping between cuisines and text search words. */
+    private static final Map<String, String[]> CUISINE_TO_SEARCH_WORDS_MAP = getCuisinesMap();
+
     /**
      * Builds a query and requests it from Google Places API.
      *
      * @param preferences the UserPreferences as specified by the user
      * @return an immutable list of places that supply the query
-     * @throws FetcherException when an error occurs in querying the Places API
-     *     for places or for places details
+     * @throws FetcherException when an error occurs in querying the Places API for
```

Comment:
Why is this in here?

Reply:
Do you mean why is this change is in this PR? Probably at some point when working on this PR it jumped to be on one line (the VScode does it for me somethimes) and when I broke the line again I broke it in a different place so it apears here as a change.

Reply:
Can you revert to how it was?

Reply:
Yes, I reverted